### DISCUSSION TITLE
测试

### DISCUSSION THREAD

**Discussion by hzjwork** [share121/inter-knot#640] [其他]
https://github.com/user-attachments/assets/0c77b295-8117-4102-a8e2-f96dc0e27750

我尝试发一个视频

**Comment by Yizakl** [comment]
它至少在我的iPhone上运行不了

**Reply by Anonymous755** [reply]
> 它至少在我的iPhone上运行不了

我的手机可以，只不过有点卡了，并且还是清朝画质

**Comment by hzjwork** [comment]
我这电脑倒是可以，就是排版有点问题，需要拖一下滑块

**Comment by 75020-maker** [comment]
还在go还在go

**Comment by Mooshed88-a** [comment]
感觉加载有点卡...毕竟Github在大陆的速度慢得像蜗牛一样。不知道支不支持插入托管平台的视频
